lighthouse-core/report/html/report-styles.css

```diff
@@ -785,10 +785,15 @@ details, summary {
   border: 0;
 }
 
-.lh-scores-header .lh-gauge__wrapper {
+.lh-scores-header .lh-gauge__wrapper,
+.lh-scores-header .lh-gauge--pwa__wrapper, {
   margin: 0 4px;
 }
 
+.lh-scores-header .lh-gauge--pwa__wrapper {
```

Comment:
it would be nice if this could be `.lh-gauge__wrapper:last-of-type`, but it doesn't work on classes. Open to other ways to do this that don't rely on the pwa wrapper being where it is

Reply:
should we really care if it's an .lh-gauge?
couldn't we just do `.lh-scores-header > :last-child` ?

Reply:
> couldn't we just do .lh-scores-header > :last-child ?

we can't because when you don't run the PWA category you don't want a vertical line before the *next* `last-child` in line once the PWA gauge is gone :) 

Maybe we should make an explicit divider line class, but this seems fine for now (if/when we ever get another special score gauge we can reevaluate)

Reply:
Of course :) forgot about running without pwa.